src/xpk/main.py

```diff
@@ -66,14 +65,8 @@ def main() -> None:
   xpk_print('Starting xpk', flush=True)
   main_args = parser.parse_args()
   main_args.enable_ray_cluster = False
-  skip_validation = 'skip_validation' in main_args and main_args.skip_validation
   dry_run = 'dry_run' in main_args and main_args.dry_run
   set_dry_run(dry_run)
-  if dry_run or skip_validation:
-    xpk_print('Skipping dependency validation.', flush=True)
```

Comment:
I'd keep the prints, so we can notice if we forget to add the validation logic when adding a new command, WDYT?